AppSettings/AppSettings/AppSettingsLoader.cs

```diff
@@ -93,7 +93,7 @@ public sealed class AppSettingsLoader
                     }
 
                     // Load arraylist setting
-                    if (IsArrayListSettingType(member))
+                    if (IsSupportedArrayListType(member))

```

Comment:
Not sure on the ArrayList name. These are basically things that support System.Collections.Generic.ICollection`1, right?